docs/cloud-native-security/environment-variable-capture.asciidoc

```diff
@@ -0,0 +1,30 @@
+[[environment-variable-capture]]
+
+== Environment variable capture
+
+.Requirements
+[sidebar]
+--
+* This feature requires {stack} version 8.6 or higher.
+* In {stack} version 8.6, this feature is only available for Linux.
+--
+
+You can configure an {agent} policy to capture up to five environment variables ("env vars"). Captured env vars are associated with process events and recorded in the `process.env_vars` field.
+
+NOTE: Env var names must be 63 characters or fewer, and env var values must be 1023 characters or fewer. Values outside these limits are silently ignored.
+
+=== Setup
+
+To set up environment variable capture for an agent policy:
+
+. Go to **Manage -> Policies**
+. Select an {agent} policy
+. Click *Show advanced settings*
+. Scroll down or search for `linux.advanced.capture_env_vars`
+. Enter the names of env vars you wish to capture, separated by commas. For example: `PATH,LD_PRELOAD,USER`
+. Click *Save* at the bottom of the page
+
+[role="screenshot"]
+image::images/env-var-capture.png[The "linux.advanced.capture_env_vars" advanced agent policy setting]
+
+NOTE: Env var names are case sensitive in Linux.
```

Comment:
This info might be helpful upfront, instead of buried at the end. Maybe combine with the other note above about character limits?